File: lab/python/binarios/binario.py
def retornar_decimal(binario):
    binario = str(binario)

    tamanho = len(binario) - 1

    soma_decimal = 0
    indice = 0
    for potencia in range(tamanho, -1, -1):
        soma_decimal += int(binario[indice]) * 2**potencia
        indice += 1
    return soma_decimal


print(retornar_decimal("10010"))

File: lab/javascript/binarios/binario.js
function retornar_decimal(binario){
    binario = String(binario);

    let tamanho = binario.length - 1;

    let soma_decimal = 0;
    let indice = 0;

    for(let potencia=tamanho; potencia > 0; potencia--){
        soma_decimal += Number(binario[indice]) * 2**potencia;
        indice ++;
    }
    return soma_decimal;
}


console.log(retornar_decimal("10010"))
